frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/LegalValueLabel/LegalValueLabel.styles.ts

```diff
@@ -4,7 +4,7 @@ export const StyledContainer = styled('div')(({ theme }) => ({
     display: 'inline-block',
     wordBreak: 'break-word',
     padding: theme.spacing(0.5, 1),
-    background: theme.palette.common.white,
```

Comment:
I guess we seldom wants to see this value used?

Reply:
Indeed, this value might not be as useful any longer